modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/security/SecurityConfig.java

```diff
@@ -196,7 +194,7 @@ private void successHandler(
      * @param ignite Ignite.
      */
     @Bean
-    public SessionRepository<ExpiringSession> sessionRepository(@Autowired Ignite ignite) {
+    public IgniteSessionRepository sessionRepository(@Autowired Ignite ignite) {
```

Comment:
Why you change this?

Reply:
Because other wise it will not injected by Spring in tests.

Reply:
Reworked to use `FindByIndexNameSessionRepository`